tests/romtools/test_vector_space_utils/test_orthogonalizer.py

```diff
@@ -19,6 +34,16 @@ def test_euclidean_l2_orthogonalizer():
   should_be_eye = my_orthogonalized_basis.transpose() @ my_orthogonalized_basis
   assert(np.allclose(should_be_eye,np.eye(2)))
 
+@pytest.mark.mpi(min_size=3)
+def test_euclidean_l2_orthogonalizer_mpi():
+  comm = MPI.COMM_WORLD
+  orthogonalizer = EuclideanL2Orthogonalizer()
+  basis_shape = (10,2)
+  local_basis, global_basis = test_utils.generate_random_local_and_global_arrays_impl(basis_shape, comm=comm)
+  local_orthogonalized_basis = orthogonalizer.orthogonalize(local_basis)
+  should_be_eye = local_orthogonalized_basis.transpose() @ local_orthogonalized_basis
+  assert(np.allclose(should_be_eye,np.eye(2)))
```

Comment:
how can this work since the Euclideanl2 thing is using interally qr factorizaton which is run via numpy?

Reply:
"should_be_eye = local_orthogonalized_basis.transpose() @ local_orthogonalized_basis"
this operation is purely localy one, you are not exercising any MPI

Reply:
Fixed